src/TextForEvent.js

```diff
@@ -475,6 +475,10 @@ function textForWidgetEvent(event) {
     const {name: prevName, type: prevType, url: prevUrl} = event.getPrevContent();
     const {name, type, url} = event.getContent() || {};
 
+    if (type === 'jitsi' || prevType === 'jitsi') {
```

Comment:
Type can also be `m.jitsi`. We have a WidgetType class somewhere which does the matching for you

Reply:
huh :/ Thanks.